workos/resources/mfa.py

```diff
@@ -115,3 +115,51 @@ def to_dict(self):
         verification_response_dict = super(WorkOSChallengeVerification, self).to_dict()
 
         return verification_response_dict
+
+
+class WorkOSAuthenticationChallengeAndFactor(WorkOSBaseResource):
```

Comment:
i would recommend naming this something like `EnrollAuthFactorResponse` instead (existing example: https://github.com/workos/workos-python/blob/fb0e087584c47f3fdc3ac29e137733b59fe2a15a/workos/resources/password_challenge_response.py#L5 )

or, leaving as two separate classes and building the response in `enroll_auth_factor` (this is what the node SDK does: https://github.com/workos/workos-node/blob/c46747ba60114bed4e0c091a3df7e97ce5a6c894/src/user-management/user-management.ts#L274)

Reply:
I went with the Node example 👍

Reply:
beautiful 🎨